uw-frame-components/portal/search/partials/search.html

```diff
@@ -7,7 +7,7 @@
             aria-label="Search bar enter the app you are looking for"
             focus-me="showSearchFocus"
             ng-show="$storage.typeaheadSearch"
-            typeahead="title as match.title for match in filterMatches | limitTo:10"
+            uib-typeahead="title as match.title for match in filterMatches | limitTo:10"
```

Comment:
Weird.  Well, if you say we _"have to change"_ something as a dependant, then that sounds like a major version to me.